### DISCUSSION TITLE
alt.s.c DNS works again

### DISCUSSION THREAD

**Discussion by carnhofdaki** [jsarenik/bitcoin-faucet-shell#8] [Show and tell]
https://alt.signetfaucet.com/ works again now. It is using https://redirect.pizza/ now and was fixed with leaving everything as was: i.e. CNAME at [alt.s.c](https://alt.signetfaucet.com/).

Still https://signetfaucet.bublina.eu.org/ is the go-to address but it is better than before (and can be set even better with no 3rd-party limit by setting a `301` (permanent) redirect at https://alt.signetfaucet.com/ to https://signetfaucet.bublina.eu.org/ on CloudFlare level).

**Comment by jsarenik** [comment]
Both broken and re-fixed today. Thanks to @dcorral

**Comment by jsarenik** [comment]
https://alt.signetfaucet.com subdomain name is broken again. No ETA. https://signetfaucet.bublina.eu.org works fine.

**Reply by jsarenik** [reply]
Redirect Pizza is intentionally not being used anymore.

The redirect from alt.s.c to https://signetfaucet.bublina.eu.org/ would wotk if set similar to https://testsf.bublina.eu.org (as a [Cloudflare Bulk Redirect](https://developers.cloudflare.com/rules/url-forwarding/bulk-redirects/) and maybe all other Signet faucets could have a redirect there. Just brainstorming aloud. Maybe I start with what I can do: list other Signet faucets on Alt.

**Comment by jsarenik** [comment]
And it works now. Without Pizza Redirect, now solely redirected by Cloudflare. Thanks to the owner of [signetfaucet.com](https://signetfaucet.com/) domain.
